packages/f/folly/xmake.lua

```diff
@@ -12,15 +12,18 @@ package("folly")
     add_versions("2022.02.14", "6a50d4cc954f0f77efc85af231ee6b7f676a0d743c12b7080fb03fd3da3ffbf2")
     add_versions("2022.04.25", "7d5cd59613780d7d2b9c946c810bfd6b0d11ed3a8a74c5ab00d4e9de5d1b2104")
     add_versions("2022.08.29", "3adac6d4b203c2917185fd190fc04d615051cb2a4f6b988ddf3c42034efc8d4d")
+    add_versions("2023.07.03", "e49901ac87bca18c0a2f71af340d63f5573bae463fdeb0c32092c6bf45621474")
 
-    add_patches("<=2022.08.29", path.join(os.scriptdir(), "patches", "2021.06.28", "reorder.patch"), "9a6bf283881580474040cfc7a8e89d461d68b89bae5583d89fff0a3198739980")
+    add_patches("<=2023.07.03", path.join(os.scriptdir(), "patches", "2021.06.28", "reorder.patch"), "9a6bf283881580474040cfc7a8e89d461d68b89bae5583d89fff0a3198739980")
     add_patches("<=2022.08.29", path.join(os.scriptdir(), "patches", "2021.06.28", "regex.patch"), "6a77ade9f48dd9966d3f7154e66ca8a5c030ae2b6d335cbe3315784aefd8f495")
 
+    add_configs("no_exception_tracer", {description = "Exclude exception tracer, which is necessary to statically link libstdc++.", default = false, type = "boolean"})
+
     if is_plat("windows") then
         add_configs("shared", {description = "Build shared library.", default = false, type = "boolean", readonly = true})
     end
 
-    add_deps("cmake")
+    add_deps("cmake <=3.24.2")
```

Comment:
why?

Reply:
I tried to investigate the reason why Windows builds failing with CMake errors, but they're failing even with master version:
https://github.com/xmake-io/xmake-repo/pull/2297/checks?check_run_id=14889087445

So this PR (config option + version update) make no degradation.